src/pages/FlagCommentPage.js

```diff
@@ -131,7 +131,7 @@ function FlagCommentPage(props) {
             title={item.name}
             description={item.description}
             onPress={Session.checkIfActionIsAllowed(() => flagComment(item.severity))}
-            style={[styles.pt2, styles.pb4, styles.mh5, styles.ph0, styles.flexRow, styles.borderBottom]}
+            style={[styles.pt2, styles.pb4, styles.ph5, styles.flexRow]}
             furtherDetails={item.furtherDetails}
             furtherDetailsIcon={item.furtherDetailsIcon}
             hoverAndPressStyle={[styles.mh0, styles.ph5]}
```

Comment:
Now that we don't have to be tricky with hover styles for the separator, I'm pretty sure we can remove this line entirely
```suggestion
```